gnssanalysis/gn_io/igslog.py

```diff
@@ -109,21 +134,25 @@ def parse_igs_log(filename_array: _np.ndarray) -> _np.ndarray:
     with open(file_path, "rb") as file:
```

Comment:
[suggestion]
```suggestion
    with open(file_path, "rb") as file:
```
The function itself is doing as a few goals. It will be good to separate the concerns so it is doing specific goals. 
for example
```
def parse_igs_log:
   call to read igs file
   cal to find V1 or V2
   them call specific funtion to read each parts.

Reply:
I've moved the "find version number" functionality out and it is now just called